### PR TITLE
Whitespace fixes around punctuators (library)

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

File: source/basic.tex [hunk 1]
```diff
@@ -991,7 +991,7 @@
 
 \begin{codeblock}
 typedef int N;
-template<N X, typename N, template<N Y> class T> struct A;
+template<N X, class N, template<N Y> class T> struct A;
```
Comment (jwakely) [inline source/basic.tex]: @zygoloid has said elsewhere that the core parts of the standard intentionally use a mix of formatting styles to reinforce the point that whitespace (or the  choice of `class` or `typename` here) has no semantic significance. So changing things to be consistent breaks that.

**Comment by jwakely** [comment]
The library parts look good, but not the rest.

@zygoloid has said elsewhere that the core parts of the standard intentionally use a mix of formatting styles to reinforce the point that whitespace (or the  choice of `class` or `typename` in template parameter lists) has no semantic significance. So changing things to be consistent breaks that intentional inconsistency.

**Comment by tkoeppe** [comment]
You mean Issue #430? Hm, that's unfortunate. I can split the library changes out if you like those.

**Comment by jwakely** [comment]
Yes. I'll merge the library parts, but I'm not touching the rest :-)

**Comment by tkoeppe** [comment]
OK, I restricted the branch to only the library changes!

File: source/utilities.tex [hunk 2]
```diff
@@ -5421,13 +5421,13 @@
 
 \begin{itemdescr}
 \pnum
-\requires Let \tcode{CT} be \tcode{common_type<unique_ptr<T1, D1>::pointer,}
+\requires Let \tcode{CT} be \tcode{common_type<unique_ptr<T1, D1>::pointer},
```
Comment (jwakely) [inline source/utilities.tex]: This comma is part of the type and should be in code font.

Comment (tkoeppe) [inline source/utilities.tex]: Oh, right, this is a manual line break. Fixed.

**Pull request merged**